packages/astro/src/stores/external.ts

```diff
@@ -13,7 +13,7 @@ import { $clerk, $csrState, $initialState } from './internal';
  *
  * $authStore.subscribe((auth) => console.log(auth.userId))
  */
-export const $authStore = computed([$csrState, $initialState], (state, initialState) => {
+export const $authStore = batched([$csrState, $initialState], (state, initialState) => {
```

Comment:
Does this return a read only store similar to computed ?

Reply:
Yes readonly too